error: call to non-‘constexpr’ function ‘long int sysconf(int)

**error: call to non-‘constexpr’ function ‘long int sysconf(int)’
[build]  8722 |     constexpr static std::size_t sigStackSize = 32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ;

/usr/include/unistd.h:640:17: note: ‘long int sysconf(int)’ declared here
[build]   640 | extern long int sysconf (int __name) __THROW;
[build]       |                 ^~~~~~~
[build] /home/yahya/Kraken/tests/../Catch2/catch.hpp:8781:45: error: size of array ‘altStackMem’ is not an integral constant-expression
[build]  8781 |     char FatalConditionHandler::altStackMem[sigStackSize] = {};**

error reported in compiler: GCC 12

--- a/Catch2/catch.hpp
+++ b/Catch2/catch.hpp
@@ -8719,7 +8719,9 @@ namespace Catch {
 
     // 32kb for the alternate stack seems to be sufficient. However, this value
     // is experimentally determined, so that's not guaranteed.
-    constexpr static std::size_t sigStackSize = 32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ;
+    #undef MINSIGSTKSZ
+    #define MINSIGSTKSZ 16384
+    constexpr static std::size_t sigStackSize = 32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ; // FIXED
 
     static SignalDefs signalDefs[] = {
         { SIGINT,  "SIGINT - Terminal interrupt signal" },

